test/new-e2e/utils/e2e/client/docker.go

```diff
@@ -99,11 +113,34 @@ func (docker *Docker) ExecuteCommandWithErr(containerName string, commands ...st
 	return output, errOutput, err
 }
 
+// GetClient gets the [docker client].
+//
+// [docker client]: https://pkg.go.dev/github.com/docker/docker/client
 func (docker *Docker) GetClient() *client.Client {
 	return docker.client
 }
 
+// GetAgentContainerName gets the agent container name
 func (docker *Docker) GetAgentContainerName() string {
 	require.NotEmptyf(docker.t, docker.agentContainerName, "agent container not found")
 	return docker.agentContainerName
 }
+
+// GetAgentCommandRunner gets an agent that provides high level methods to run Agent commands.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
What about 
```golang
// GetAgentCommandRunner gets a runner that provides high level methods to run Agent commands.
```